index.js

```diff
@@ -103,65 +104,26 @@ class DeltaChat extends EventEmitter {
 
     this.once('_configured', ready)
 
-    this.setConfig('addr', opts.addr)
-
-    if (typeof opts.mail_server === 'string') {
-      this.setConfig('mail_server', opts.mail_server)
-    }
+    if (!opts.e2ee_enabled) opts.e2ee_enabled = 1
```

Comment:
Doing `opts.e2ee_enabled = false` will make it true. Maybe check for `undefined` instead?